.gitignore

```diff
@@ -25,3 +25,7 @@ ubuntu-*-cloudimg-console.log
 
 # Mainly generated by merge tools like 'meld'
 *.orig
+
+# IDE's project files
+.idea
+.vscode
```

Comment:
This was probably not intended to be a part of the MR.

Reply:
Yes, will be remove

Reply:
Done